receiver/receiverhelper/metricsfilter.go

```diff
@@ -0,0 +1,61 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package receiverhelper
+
+import (
+	"context"
+	"fmt"
+
+	"go.uber.org/zap"
+
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/processor/filterprocessor"
+)
+
+type FilterSettings struct {
```

Comment:
Any reason not to just use `MetricFilters` directly?

Reply:
I don't believe so, updated to do so.

Reply:
@pmcollins - had to put this back. I realized we need to wrap the config struct from `filterprocessor` so that we have a centralized config key (`metrics_filter` in this case).

Reply:
Ah, ok, sorry for the trouble. Consider adding a comment if you think it might be helpful.